workitem/number_sequence/workitem_number_sequence.go

```diff
@@ -1,12 +1,15 @@
-package workitem
+package numbersequence
 
 import (
+	"fmt"
+
 	uuid "github.com/satori/go.uuid"
 )
 
 // WorkItemNumberSequence the sequence for work item numbers in a space
 type WorkItemNumberSequence struct {
-	SpaceID    uuid.UUID `sql:"type:uuid" gorm:"primary_key"`
+	ID         uuid.UUID `sql:"type:uuid" gorm:"primary_key"`
+	SpaceID    uuid.UUID `sql:"type:uuid"`
```

Comment:
This is one of the most important changes of this PR, isn't it?

Reply:
yes, indeed